// Copyright (c) 2020-2021 The Open-Transactions developers
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at http://mozilla.org/MPL/2.0/.

syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTBlockchainEnums";
option optimize_for = LITE_RUNTIME;

enum BlockchainTransactionOutputType {
    BTOUTPUT_ERROR = 0;
    BTOUTPUT_UNKNOWN = 1;
    BTOUTPUT_P2PK = 2;
    BTOUTPUT_P2PKH = 3;
    BTOUTPUT_P2SH = 4;
    BTOUTPUT_MULTISIG = 5;
    BTOUTPUT_NULL = 6;
    BTOUTPUT_P2WPKH = 7;
    BTOUTPUT_P2WSH = 8;
}
